### PR TITLE
Integrating against usercard api so we get followers counts, etc

### PR DESCRIPTION
**Pull request by chrisnojima**
also Making protocol makefile copy the js files into the right place

@keybase/react-hackers

### CODE REVIEW

File: react-native/react/actions/tracker.js [hunk 1]
```diff
@@ -9,16 +9,14 @@ import {flattenCallMap, promisifyResponses} from '../engine/call-map-middleware'
...
-import type {UserSummary} from '../constants/types/flow-types'
 import type {Action, Dispatch} from '../constants/types/flux'
-import type {UserInfo} from '../tracker/bio.render.types'
 
-import type {Identity, IdentifyKey, TrackSummary, User, Cryptocurrency, IdentifyOutcome, RemoteProof, LinkCheckResult, TrackOptions} from '../constants/types/flow-types'
+import type {Identity, IdentifyKey, TrackSummary, User, Cryptocurrency, IdentifyOutcome, RemoteProof, LinkCheckResult, TrackOptions, UserCard} from '../constants/types/flow-types'
 
 // TODO make actions for all the call back stuff.
 
 export function registerIdentifyUi (): (dispatch: Dispatch) => void {
```
Comment (MarcoPolo) [inline react-native/react/actions/tracker.js]: this type isn't correct, but you can use the newly merged TrackerActionCreator: https://github.com/keybase/client/pull/1479/files#diff-49efa58b5e3f2f865d5c952996d692b3R18

File: react-native/react/actions/tracker.js [hunk 2]
```diff
@@ -147,45 +145,27 @@ export function onCloseFromHeader (username: string): Action {
...
-        followersCount: -1,
-        followingCount: -1,
-        followsYou: false
-      }, username))
-    })
-  }
-}
-
-function updateUserInfo (userInfo: UserInfo, username: string): Action {
+function updateUserInfo (userCard: UserCard, username: string, getState: Function): Action {
```
Comment (MarcoPolo) [inline react-native/react/actions/tracker.js]: change these to use the TrackerActionCreator type.

**Comment by MarcoPolo** [comment]
LG%types

File: react-native/react/actions/tracker.js (hunk 2 above)
Comment (chrisnojima) [inline react-native/react/actions/tracker.js]: but this doens't match right? it just returns a payload, not the arrow

Comment (MarcoPolo) [inline react-native/react/actions/tracker.js]: ah you're right sorry, then just update the getState to be `getState: () => {tracker: RootTrackerState}`

**Pull request merged**